### PR TITLE
docs: Fix some minor typos in the home template

### PR DESCRIPTION
**Pull request by profanis**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [ ] No

### CODE REVIEW

File: projects/demo/src/modules/app/home/home.template.html [hunk 1]
```diff
@@ -214,9 +214,9 @@
                 <code>SVG</code> inlining. Taiga UI icons will be automatically
                 downloaded and inlined directly into DOM. However, Angular
                 <code>Sanitizer</code> removes <code>SVG</code> code in that
-                case. To use custom sanitizer that properly supports
-                <code>SVG</code> provide <code>TUI_SANITIZER</code> token. We
-                recommend you using our library
+                case. To use a custom sanitizer that properly supports
+                <code>SVG</code> provide the <code>TUI_SANITIZER</code> token. We
+                recommend you, using our library
```
Comment (waterplea) [inline projects/demo/src/modules/app/home/home.template.html:219]: I think this should rather be changed to `...you to use our...`
Comma seems wrong here.

**Review by waterplea — CHANGES_REQUESTED** [review]

File: projects/demo/src/modules/app/home/home.template.html (hunk 1 above)
Comment (profanis) [reply projects/demo/src/modules/app/home/home.template.html:219]: I was in between your suggestion and what I wrote. I will do the change :)

Comment (profanis) [reply projects/demo/src/modules/app/home/home.template.html]: @waterplea I did the changes

**Review by MarsiBarsi — APPROVED** [review]
Thank you for your contribution! 🎉

**Pull request merged**